app/src/main/java/com/sopt/now/compose/presentation/auth/login/LoginViewModel.kt

```diff
@@ -0,0 +1,32 @@
+package com.sopt.now.compose.presentation.auth.login
+
+import android.util.Log
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.sopt.now.compose.model.login.RequestLoginDto
+import com.sopt.now.compose.model.login.ResponseLoginDto
+import com.sopt.now.compose.utils.ServicePool.loginService
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class LoginViewModel: ViewModel() {
+    private var _login = MutableLiveData<ResponseLoginDto>()
+    var login : MutableLiveData<ResponseLoginDto> = _login
```

Comment:
backing property사용 좋습니다!!
아래 방법과 어떤 차이가 있는지 보시면 좋습니다! xml에 상세하게 달아놓았어요
```suggestion
    private val _login = MutableLiveData<ResponseLoginDto>()
    val login : MutableLiveData<ResponseLoginDto> get() = _login
```